### DISCUSSION TITLE
Understanding .glb model properly

### DISCUSSION THREAD

**Discussion by ChristofJugel** [google/model-viewer#3070] [Q&A]
Hello :) 

Does any of you have a documentation on how the 3D models work? 

I don't really understand how to animate color or add variants.

Best

**Comment by elalish** [comment]
Generally I would recommend using animation software that exports to glTF like Blender or any of the big expensive ones. If you really want to generate them by hand or programmatically, I'd recommend the glTF VSCode  extension and you can get started with the spec  here: https://www.khronos.org/files/gltf20-reference-guide.pdf
